app/components/post_list/post_list.js

```diff
@@ -315,6 +318,20 @@ export default class PostList extends PureComponent {
         }
     };
 
+    scrollToInitialIndexIfNeeded = (index, count = 0) => {
+        if (!this.hasDoneInitialScroll && this.flatListRef?.current) {
+            this.hasDoneInitialScroll = true;
+
+            if (index > 0 && index <= this.getItemCount()) {
+                this.scrollToIndex(index);
+            } else if (count < 3) {
```

Comment:
what is the significance if count 3 here?

Reply:
I wanted to retry 3 times before giving up